src/dataflow_pipeline.py

```diff
@@ -253,16 +253,20 @@ def cdc_apply_changes(self):
             else self.silver_schema
         )
 
+        sequenced_by_data_type = None
+
```

Comment:
where it's checked that `cdc_apply_changes.sequence_by` exists? if you don't have it, then `sequenced_by_data_type` won't be initialized...

Reply:
its checking during onboarding process as mandatory field: [__validate_apply_changes](https://github.com/databrickslabs/dlt-meta/blob/issue/apply_changes_infer_version_col_datatype/src/onboard_dataflowspec.py#L510):